docs/azure-simple/README.md

```diff
@@ -318,7 +318,7 @@ service_principal_id = "7b6ab9ae-dead-abcd-8b52-0a8ecb5beef7"
 service_principal_secret = "35591cab-13c9-4b42-8a83-59c8867bbdc2"
 ssh_public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCo5cFB/HiJB3P5s5kL3O24761oh8dVArCs9oMqdR09+hC5lD15H6neii4azByiMB1AnWQvVbk+i0uwBTl5riPgssj6vWY5sUS0HQsEAIRkzphik0ndS0+U8QI714mb3O0+qA4UYQrDQ3rq/Ak+mkfK0NQYL08Vo0vuE4PnmyDbcR8Pmo6xncj/nlWG8UzwjazpPCsP20p/egLldcvU59ikvY9+ZIsBdAGGZS29r39eoXzA4MKZZqXU/znttqa0Eed8a3pFWuE2UrntLPLrgg5hvliOmEfkUw0LQ3wid1+4H/ziCgPY6bhYJlMlf7WSCnBpgTq3tlgaaEHoE8gTjadKBk6bcrTaDZ5YANTEFAuuIooJgT+qlLrVql+QT2Qtln9CdMv98rP7yBiVVtQGcOJyQyG5D7z3lysKqCMjkMXOCH2UMJBrurBqxr6UDV3btQmlPOGI8PkgjP620dq35ZmDqBDfTLpsAW4s8o9zlT2jvCF7C1qhg81GuZ37Vop/TZDNShYIQF7ekc8IlhqBpbdhxWV6ap16paqNxsF+X4dPLW6AFVogkgNLJXiW+hcfG/lstKAPzXAVTy2vKh+29OsErIiL3SDqrXrNSmGmXwtFYGYg3XZLiEjleEzK54vYAbdEPElbNvOzvRCNdGkorw0611tpCntbpC79Q/+Ij6eyfQ== user"
 gitops_ssh_url = "git@github.com:<user>/bedrock-deploy-demo.git"
-gitops_ssh_key = "/home/<user>/.ssh/azure-simple-deploy-key"
+gitops_ssh_key = "/home/<user>/.ssh/gitops-ssh-key"
```

Comment:
Did this require an absolute path when you did this during the hack (I can't remember)?   If not, could you adjust to be "~/.ssh/gitops-ssh-key" to avoid having to lock the script down to just the current user.

Reply:
Unfortunately yes, you need the absolute path.  This was the error I got when trying to use the relative path
```bash 
creating kubernetes secret flux-ssh from key file path ~/.ssh/azure-simple-deploy-key
error: error reading ~/.ssh/azure-simple-deploy-key: no such file or directory
ERROR: failed to create secret: flux-ssh
```
And just verifying that the keys are there.
```bash
$ ls ~/.ssh/azure-simple-deploy-key*
/home/matt/.ssh/azure-simple-deploy-key  /home/matt/.ssh/azure-simple-deploy-key.pub
```